builds/configuring-build-strategies.adoc

```diff
@@ -0,0 +1,38 @@
+:_content-type: ASSEMBLY
+[id="configuring-build-strategies"]
+= Configuring build strategies 
+:context: configuring-build-strategies
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+
+You can configure build strategies by defining strategy parameters, system parameters, steps resource definition, annotations, and volumes in a `BuildStrategy` or `ClusterBuildStrategy` custom resource (CR).
+
+
+include::modules/ob-defining-strategy-parameters.adoc[leveloffset=+1]
+
+include::modules/ob-defining-system-parameters.adoc[leveloffset=+1]
+
+include::modules/ob-defining-step-resources.adoc[leveloffset=+1]
+
+include::modules/ob-strategies-with-different-resources.adoc[leveloffset=+2]
+
+include::modules/ob-resource-management-in-tekton-pipelines.adoc[leveloffset=+2]
+
+include::modules/ob-examples-of-tekton-resources-management.adoc[leveloffset=+3]
+
+include::modules/ob-defining-annotations.adoc[leveloffset=+1]
+
+include::modules/ob-securely-referencing-string-parameters.adoc[leveloffset=+1]
+
+include::modules/ob-defining-system-results.adoc[leveloffset=+1]
+
+include::modules/ob-defining-volumes-and-volume-mounts.adoc[leveloffset=+1]
+
+
+
+// [role="_additional-resources"]
+// == Additional resources
+
+// * For information on how to use the strategy parameters in a `Build` or `BuildRun` CR, see 
```

Comment:
If this additional resources section ends up being added, please format it per the guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

Reply:
Sure, @abrennan89. Since the content is being created from scratch, I do not have all the references ready.